client/src/sections/resources/Cartography.tsx

```diff
@@ -0,0 +1,166 @@
+import { ListItem, UnorderedList } from "@chakra-ui/react";
+import Line from "../../components/Line";
+import Tile from "../../components/Tile";
+import { useMyLevel } from "../../hooks/useCall";
+import useGet from "../../hooks/useGet";
+import useHave from "../../hooks/useHave";
+import { $skill } from "../../util/makeValue";
+import { plural } from "../../util/text";
+
+// ==== TILE TO-DO LIST ==================
+//   - Want to figure out a good way to make the bullets into links to the respective zones or containers.
+//   - Add more map targets with more conditions on them. (If you have a spit up, you probably want to map a good spit dude, for instance.)
+//   - Better ways to re-order & re-prioritize the map targets.
+//   - Can we do a for loop within the <Tile> to enumerate mapTarget? That would be so much better if it is at all possible.
+
+/**
+ * Class used to store map targets. Unless otherwise specified, all params are strings.
+ * @param monster The monster in question
+ * @param zone The zone you'll find the monster in
+ * @param level The level this map target is feasible (number)
+ * @param access A way to pass in a custom condition for accessibility (boolean)
+ */
+
+class MapTarget {
+  monster: string;
+  zone: string;
+  level: number;
+  access: boolean;
+  turnsTilGROPs: number;
+
+  constructor(
+    monster: string,
+    zone: string,
+    level: number,
+    access?: boolean,
+    turnsTilGROPs?: number
+  ) {
+    this.monster = monster;
+    this.zone = zone;
+    this.level = level;
+    this.access = access ?? true;
+    this.turnsTilGROPs = turnsTilGROPs ?? 0;
+  }
+
+  /**
+   * Returns true/false depending on if you are at a level high enough to access this monster.
+   * @param monster The monster in question
+   */
+  accessible(userLevel: number): boolean {
+    return this.access && userLevel > this.level;
+  }
+
+  /**
+   * Returns a string for the mapTarget to feed into your <Line> statement.
+   */
+  formatList(userLevel: number): React.ReactNode {
+    // We want to output special text for GROP availability. There are two possible states:
+
+    //   - Grops are available, in which case it just does exactly what the bullets normally do.
+    //   - Grops are available in X turns, in which case it just lets you know they'll be ready in X turns.
+
+    if (!this.accessible(userLevel)) {
+      return <></>;
+    }
+
+    if (this.monster === "Green Ops Soldier") {
+      return (
+        <ListItem ml="3" fontSize="sm">{`Green Ops Soldier
+          ${
+            this.turnsTilGROPs > 0
+              ? `(possible in ${plural(this.turnsTilGROPs, "war turn")})`
+              : ` @ ${this.zone}`
+          }`}</ListItem>
+      );
+    } else {
+      return (
+        <ListItem
+          ml="3"
+          fontSize="sm"
+        >{`${this.monster} @ ${this.zone}`}</ListItem>
+      );
+    }
+  }
```

Comment:
sorry, I changed my mind about how this should be styled - can you make this a new component in just this file, `<MapTargetItem key={target.monster} target={target} />`? When you have a `map` result in a list of react nodes, you need to give each a `key` prop so React can keep track of them.

You should be able to get rid of the `ml` and `fontSize` props now.